src/frontend/src/components/creacionCuentas/ctaCorriente.vue

```diff
@@ -71,9 +81,25 @@
   margin-left: 1rem;
   margin-right: 2rem;
   margin-top: 10px;
+  background-color: red;
+  border-radius: 10px;
+  color: #ffffff;
 }
 
 h2 {
   padding: 1rem;
 }
```

Comment:
Sugerencia: cambiar el color del texto junto a la imágen a color azul.

```suggestion
h2 {
  padding: 1rem;
  color: #0039A6;
}
```